servicemodels/test_sm_aper_go_lib/v1/test-sm-ies/test_sm.pb.go

```diff
@@ -262,10 +272,8 @@ type TestConstrainedInt struct {
 	AttrCiD int32 `protobuf:"varint,4,opt,name=attr_ci_d,json=attrCiD,proto3" json:"attr_ci_d,omitempty" aper:"valueUB:100"`
 	// @inject_tag: aper:"valueLB:10,valueUB:20"
 	AttrCiE int32 `protobuf:"varint,5,opt,name=attr_ci_e,json=attrCiE,proto3" json:"attr_ci_e,omitempty" aper:"valueLB:10,valueUB:20"`
-	// @inject_tag: aper:"valueLB:10,valueUB:10"
-	AttrCiF int32 `protobuf:"varint,6,opt,name=attr_ci_f,json=attrCiF,proto3" json:"attr_ci_f,omitempty" aper:"valueLB:10,valueUB:10"`
-	// @inject_tag: aper:"valueLB:10,valueUB:10,valueExt"
-	AttrCiG int32 `protobuf:"varint,7,opt,name=attr_ci_g,json=attrCiG,proto3" json:"attr_ci_g,omitempty" aper:"valueLB:10,valueUB:10,valueExt"`
+	// @inject_tag: aper:"valueLB:10"
```

Comment:
Is this file updated intentionally or is it a left over from 'make protos'? In case of the latter, please exclude it from the PR